test/unit/player.test.js

```diff
@@ -59,35 +58,31 @@ test('should create player instance that inherits from component and dispose it'
 });
 
 test('should accept options from multiple sources and override in correct order', function(){
-  // For closure compiler to work, all reference to the prop have to be the same type
-  // As in options['attr'] or options.attr. Compiler will minimize each separately.
-  // Since we're using setAttribute which requires a string, we have to use the string
-  // version of the key for all version.
 
   // Set a global option
-  globalOptions['attr'] = 1;
+  videojs.options.attr = 1;
```

Comment:
Potentially change this to player prototype options since this is the player tests.